NEditor.js

```diff
@@ -114,8 +114,8 @@ NEditor.onNodeDragMouseUp = function(e){
 NEditor.onNodeDragMouseMove = function(e){
 	e.stopPropagation(); e.preventDefault();
 	if(NEditor.dragItem){
-	  NEditor.dragItem.style.left = e.pageX + NEditor.offsetX;
-	  NEditor.dragItem.style.top = e.pageY + NEditor.offsetY;
```

Comment:
If I am not mistaken is this the only change to the codebase other than deleting whitespace and formatting?